clients/cobol-lsp-vscode-extension/src/test/suite/testHelper.ts

```diff
@@ -259,3 +261,44 @@ export async function executeCommandMultipleTimes(
     await vscode.commands.executeCommand(command);
   }
 }
+
+export async function getWorkspaceFile(workspace_file: string) {
+  const files = await vscode.workspace.findFiles(workspace_file);
+
+  assert.ok(files && files[0], workspace_file);
+
+  return files[0];
+}
+
+export async function waitForDiagnosticsChange(file: string | vscode.Uri) {
```

Comment:
What is so different/better about it?

Reply:
you can do ```await waitForDiagnosticsChange(file, ()=> {action();});```
instead of 
```
const diagPromise = waitForDiagnosticsChange();
/*await*/ action();
await diagPromise;
```

Reply:
I saw that, but it felt strange to me to add callbacks into async code.